script.js

```diff
@@ -0,0 +1,18 @@
+// -----------------Storing information in Js object --------------------------------
+const project = {
+  name: 'portfolio',
+  description:
+    'Personal portfolio that contains project information, skills, and contact',
+  technologies: ['HTML', 'CSS', 'JavaScript', 'Bootstrap'],
+  liveVersion: 'https://ridabensalem.github.io/',
+  sourceCode: 'https://github.com/ridabensalem/ridabensalem.github.io',
+};
+//  -----------------Handiling the pop up functionality----------------------------
+const buttonPopup = document.querySelector('button');
+const modalPopup = document.querySelector('.block-popup');
+function Display() {
+  modalPopup.style.display = 'block';
+}
+function closePopup() {
```

Comment:
While you did a good job trying to work on the requirements I noticed your pop-up window both in your desktop and mobile view does not match the figma template it would be nice if you carefully inspect the template you picked.

This is your design when i click the see project button for both your mobile and desktop view:

![Desktop snip](https://user-images.githubusercontent.com/79506105/210756864-7f5cfd6e-fce2-4127-8db8-bdd3fbaee4a9.PNG)
![mobile snip1](https://user-images.githubusercontent.com/79506105/210756905-4a63604a-0bea-49c7-b0a9-255662f5dfff.PNG)

Kindly stick to this figma design:

![mobile snip figma](https://user-images.githubusercontent.com/79506105/210756972-7ffcea46-5ea0-4bbe-8bb9-ff43e5faa072.PNG)
![figma desktop snip](https://user-images.githubusercontent.com/79506105/210756990-3bba7991-5d45-419b-b3ae-297405ca10c0.PNG)
 Thats how your pop up window should look when the see project button is clicked without zooming in or out, Please kindly fix this issue